react/components/LogoutCustomerSession.tsx

```diff
@@ -49,19 +53,39 @@ interface Props {
 const LogoutCustomerSession = (props: Props) => {
   const { client, loading, onDepersonify, attendantEmail, mobile } = props
   const handles = useCssHandles(CSS_HANDLES)
+  const { orderForm } = useOrderForm()
 
-  const getClientName = (client: any) =>
-    !!client
-      ? client.name.includes('null')
-        ? client.email.slice(0, client.email.indexOf('@'))
-        : client.name
+  const { id } = orderForm
+
+  const [customerClass, setCustomerClass] = useState('')
+
+  useEffect(()=> {
+    function getCustomerClass() {
+      const requestOptions = {
+        method: 'GET',
+      }
+      return fetch(`/api/checkout/pub/orderForm/${id}`, requestOptions)
+    }
+  
+    getCustomerClass().then(response => response.json()).then(data => {
+      if(!!data?.clientProfileData?.customerClass) {
```

Comment:
```suggestion
      if (!!data?.clientProfileData?.customerClass) {
```

Reply:
@igorbrasileiro Updated the code as per suggestion